### ISSUE TITLE
Collector cron warning

### ISSUE THREAD

**Issue by pVesian** [glpi-project/glpi#1607]
Hi,

On 9.1.1, after setting up a functionning mail collector, I found this in the PHP error log:

```
*** PHP Notice(8): Undefined index: glpi_dropdowntranslations
  Backtrace :
  inc/notificationtemplate.class.php:230             
  inc/notificationevent.class.php:168                NotificationTemplate->getTemplateByLanguage()
  inc/ticket.class.php:1775                          NotificationEvent::raiseEvent()
  inc/commondbtm.class.php:853                       Ticket->post_addItem()
  inc/mailcollector.class.php:564                    CommonDBTM->add()
  inc/mailcollector.class.php:1614                   MailCollector->collect()
  inc/crontask.class.php:834                         MailCollector::cronMailgate()
  front/cron.php:88                                  CronTask::launch()
```

I don't really know how I got it, so I don't know how you can reproduce it. The collector is a gmail inbox running on IMAP and works fine.

Setup:

```
Operating system: Linux model-ubuntu 3.13.0-30-generic #54-Ubuntu SMP Mon Jun 9 22:47:59 UTC 2014 i686
PHP 7.0.14-1~dotdeb+8.1 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, calendar,
	ctype, curl, date, dom, exif, fileinfo, filter, ftp, geoip, gettext, gmp, hash, iconv, imap, intl, json, ldap, libxml, mbstring,
	mcrypt, msgpack, mysqli, mysqlnd, openssl, pcre, pdo_mysql, pdo_sqlite, posix, readline, session, shmop, soap, sockets, sqlite3,
	standard, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xml, xmlreader, xmlrpc, xmlwriter, xsl, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M" 
Software: Apache/2.4.7 (Ubuntu) (Apache/2.4.7 (Ubuntu) Server at localhost Port 80)
	Mozilla/5.0 (X11; Ubuntu; Linux i686; rv:30.0) Gecko/20100101 Firefox/30.0
Server Software: (Ubuntu)
	Server Version: 5.5.35-1ubuntu1
	Server SQL Mode: 
	Parameters: root@localhost/glpi
	Host info: Localhost via UNIX socket
```

Thanks!

**Comment by AnaelMobilia** [comment]
Hello,

This issue have been solved on #1588 (on the dev version ;-))

Thanks for the report !

**Comment by trasher** [comment]
Thanks @AnaelMobilia!
